### PR TITLE
Implement Styling and Starter Files for Hooks

### PR DESCRIPTION
**Pull request by yozzyurubuza**
Add new files and basic component codes in preparation for studying and implementing hooks.

### CODE REVIEW

File: 12-hooks-in-react/src/components/Accordion.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import React from "react";
+
+const Accordion = ({ items }) => {
+  const renderedItems = items.map((item) => {
+    return (
+      <React.Fragment key={item.title}>
```
Comment (jcgalbano) [inline 12-hooks-in-react/src/components/Accordion.js:6]: Title is not a good key as if 2 or more titles are the same this'll error out, used the index of the items instead or make your own counter for this that's unique.

**Review by jcgalbano — CHANGES_REQUESTED** [review]
Good job just a few suggestions

File: 12-hooks-in-react/public/index.html [hunk 2]
```diff
@@ -24,6 +24,11 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
+    <link
```
[the remark this answers is not in the document]
Comment (yozzyurubuza) [reply 12-hooks-in-react/public/index.html:27]: Noted on this. Thanks!

File: 12-hooks-in-react/src/components/Accordion.js (hunk 1 above)
Comment (yozzyurubuza) [reply 12-hooks-in-react/src/components/Accordion.js]: Implemented the change of key prop to index. Please check my latest commit.

**Review by jcgalbano — APPROVED** [review]

**Comment by jcgalbano** [comment]
lgtm

**Pull request merged**